code/src/components/PopularList.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen,Ubuntu,Cantarell,Fira Sans,Droid Sans,Helvetica Neue,sans-serif;
```

Comment:
Woah that's a lot of fallbacks fonts. Are you implementing the Google Fonts fonts that you are using here somewhere?